store/tx_manager.go

```diff
@@ -270,37 +271,41 @@ func (txm *EthTxManager) GetLINKBalance(address common.Address) (*assets.Link, e
 	return (*assets.Link)(balance), nil
 }
 
-// MeetsMinConfirmations returns true if the given transaction hash has been
+// BumpGasUntilSafe returns true if the given transaction hash has been
 // confirmed on the blockchain.
-func (txm *EthTxManager) MeetsMinConfirmations(hash common.Hash) (bool, error) {
-	if !txm.Connected() {
-		return false, ErrPendingConnection
-	}
-	blkNum, err := txm.GetBlockNumber()
-	if err != nil {
-		return false, err
-	}
-	attempts, err := txm.getAttempts(hash)
+func (txm *EthTxManager) BumpGasUntilSafe(hash common.Hash) (*TxReceipt, error) {
+	blkNum, err := txm.getBlockNumber()
```

Comment:
Perhaps we could rename this to `safeGetBlockNumber` or something so it's not so easily mistaken with the original method?